web/modules/custom/weather_block/src/Plugin/Block/WeatherBlock.php

```diff
@@ -138,7 +134,7 @@ public function blockForm($form, FormStateInterface $form_state) {
       '#collapsed' => FALSE,
     ];
 
-    $form['city_selection']['city'] = [
+    $form['city_selection']['default_city'] = [
       '#type' => 'select',
       '#title' => $this->t('Choose a city'),
```

Comment:
You changed the key for the field
For a user, it is still unclear what that city is.
Please pay attention to your code
Please review your changes.